ledger/error/src/main/scala/com/daml/error/definitions/ErrorGroups.scala

```diff
@@ -13,11 +13,15 @@ object ErrorGroups {
       abstract class DatabaseErrorGroup extends ErrorGroup()
     }
     abstract class LedgerApiErrorGroup extends ErrorGroup() {
-      abstract class CommandExecutionErrorGroup extends ErrorGroup()
-      abstract class PackageServiceErrorGroup extends ErrorGroup()
       abstract class AdminServicesErrorGroup extends ErrorGroup() {
         abstract class UserManagementServiceErrorGroup extends ErrorGroup()
       }
+      abstract class AuthorizationChecks extends ErrorGroup()
+      abstract class CommandExecutionErrorGroup extends ErrorGroup()
+      abstract class ConsistencyErrors extends ErrorGroup()
+      abstract class PackageServiceErrorGroup extends ErrorGroup()
+      abstract class RequestValidation extends ErrorGroup()
+      abstract class WriteServiceRejections extends ErrorGroup()
```

Comment:
This is a good place to define how error groups are nested as it's uncluttered with error code and error definitions